src/shared/backgroundScripts/getters.js

```diff
@@ -27,6 +29,10 @@ export async function getTelemetryEnabled() {
     return result.telemetryEnabled;
   }
 }
+
+
+export const isFirefoxExtension = IS_FIREFOX_EXTENSION;
```

Comment:
I noticed in this patch that you inverted many if-else conditions due to the logically inverted meaning of the boolean.

If it makes sense, you could consider introducing a boolean for the Chromium extension as well.

If it makes it easier, imagine there to be a third browser. What condition would you use?

Reply:
Good point. I imagine for arbitrary amounts of browser types it would be more appropriate to use the environment string directly in a var like `browserType` instead of a Boolean, then regular if-else or case switching against the string. 

This reminds me of a question I had. I'm no expert in the shipping aspect of extensions:

Would the implication of using a `browserType` string instead mean we should be building per chromium browser as well? I imagine one chromium build would be preferred (forced?) for unification. Had this thought when looking for the best way to log which web store the extension was installed from.

Reply:
I wouldn't use strings, just readable booleans. Being Chromium is not mutually exclusive with being another browser. E.g. in some extensions that I develop, I have something like `IS_CHROMIUM`, which is true for Chrome and Chromium-based such as Opera (since Chromium extension APIs are mostly the same across Chromium forks), and then a separate `IS_OPERA` for Opera-specific stuff.

In my initial review I suggested to use one separate module instead of looking up `minimum_chrome_version` everywhere. That was to minimize the amount of code changes needed across the code base and/or build script. Since your implementation uses DefinePlugin, you don't actually need a separate module, but you could also use that defined variable directly. The advantage of that over the module export is that an inline compile-time constant boolean can allow the code generator to eliminate dead code (e.g. exclude Firefox-specific code in the build for the Chrome extension).

Reply:
In the changes I've submitted, I added in the chromium boolean and used it for readability purposes (since it works the same). I did not change the ternary statements, but they are not necessarily as readable as the if elses. Should I be converting the platform-checking ternarys into better readable if-else statements?

Also, interesting about the dead code elimination in webpack, did not know that existed